### PR TITLE
Stress testing v1

### PR DESCRIPTION
**Pull request by masoncpott**
@sam-masno @sjamcox @tyler-thee-creator 

Installed New Relic.
Installed artillery as a dev dependency.
Created a .yml file to write a 4 phase stress test.
Conducted stress test, captured screenshots.

### CODE REVIEW

File: checkout-service/stressTest.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+      arrivalRate: 10
+      name: "Ten users per second for 60 seconds"
+    - pause: 10
+    - duration: 60
+      arrivalRate: 100
+      name: "100 users per second for 60 seconds"
+    - pause: 10
+    - duration: 60
+      arrivalRate: 1000
+      name: "1000 users per second for 60 seconds"
```
Comment (samasno) [inline checkout-service/stressTest.yml:18]: good ramping setup

File: checkout-service/stressTest.yml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+      - "999"
+      - "9999"
+      - "99999"
+      - "999999"
+      - "9999999"
+      - "888"
+      - "8888"
+      - "88888"
+      - "888888"
+      - "8888888"
```
Comment (samasno) [inline checkout-service/stressTest.yml:30]: to build more randomness with the ids you're requesting you can use the config.processor and get.beforeRequest hook to use a js javascript function to automate

**Review by samasno — APPROVED** [review]
solid start on load testing

File: checkout-service/stressTest.yml (hunk 2 above)
Comment (masoncpott) [reply checkout-service/stressTest.yml:30]: I'll look into this! Thanks!

**Pull request merged**